### ISSUE TITLE
CSS Reveals?

### ISSUE THREAD

**Issue by TCEmkenn** [Third-Coast/website#38]
The staff is all noticing that when the main pages load on the website (the home page, the Competition Page, the Conference Page, the Residency Page), you can see the headers load really slowly, in real time! Any idea what's happening here/why it's so slow?

**Comment by brendanmetzger** [comment]
This may be related to #31 I've noticed, but I can't say for sure, I've experienced the flashing on first load, which is related to fonts (again, #31) but nothing crazy slow. Does anyone have enough time to take a quick screenshot so I can see the particulars?

**Comment by brendanmetzger** [comment]
I'm getting a very slow load in safari, and I think it has to do with all the fonts. Still investigating.

**Comment by TCEmkenn** [comment]
OK sounds good! I'm on Chrome, and getting a slow load. Is it still helpful
for me to send a screenshot?

*Emily S. Kennedy* | Program Director | Third Coast Festival
<http://thirdcoastfestival.org/>

ekennedy@thirdcoastfestival.org | Pronouns: She/Her/Hers

**Comment by brendanmetzger** [comment]
Yeah, a screenshot would help if at all possible… I haven’t had anything REALLY slow yet, so I’m not quite sure what’s up

**Comment by TCEmkenn** [comment]
Ugh, I am trying and failing to take a screenshot! It's worst for me on the Residency page, but it's happening on all of the pages (even the front page).
